src/autoscaler/eventgenerator/generator/evaluator.go

```diff
@@ -71,7 +71,7 @@ func (e *Evaluator) doEvaluate(triggerArray []*model.Trigger) {
 		}
 		for _, appMetric := range appMetricList {
 			if appMetric.Value == nil {
-				e.logger.Info("should not send trigger alarm to scaling engine because there is nil-value metric", lager.Data{"trigger": trigger, "appMetric": appMetric})
+				e.logger.Info("should not send trigger alarm to scaling engine because there is nil-value metric( When aggrgator gets an empty metric array from metric-collector in a statWindow,it creates an AppMetric instance with nil Value)", lager.Data{"trigger": trigger, "appMetric": appMetric})
```

Comment:
Huh?
Did you really mean to do this?
Again, what does this mean to the operator? When they see this message, what are they supposed to do?
Why is this one Info and the others Debug?